docs/topics/files.txt

```diff
@@ -55,6 +55,10 @@ it has all the methods and attributes described below.
     file name used on disk cannot be relied on until after the model has been
     saved.
 
+.. note::
```

Comment:
I think it would be more interesting to show this through an example by continuing the code block example above and showing the use case for doing `Image.open(instance.image)`.